larray/inout/common.py

```diff
@@ -18,6 +18,8 @@
 _supported_scalars_types = (int, float, bool, basestring, date, time, datetime)
 _supported_types = _supported_larray_types + _supported_scalars_types
 _supported_typenames = {cls.__name__ for cls in _supported_types}
+if PY2:
+    _supported_typenames.add('str')
```

Comment:
This seems fishy. I think the problem is that we should not use basestring in this context (or maybe any context?) and use bytes and unicode individually instead.

Reply:
why not just replace basestring by str? It should be enough, right?

Reply:
because then you don't support the same thing on Python2 and Python3, str on python2 is equivalent to bytes on Python3. That's why I would have used "unicode" instead (until we drop support for Python2)